src/amo/components/SearchContextCard/index.js

```diff
@@ -25,10 +31,27 @@ export class SearchContextCardBase extends React.Component {
   render() {
     const { count, filters, i18n, loading } = this.props;
     const { query } = filters;
+    const { addonType } = filters;
 
     let searchText;
+    let addOn;
 
-    if (!loading && query) {
+    if (addonType === ADDON_TYPE_EXTENSION) {
+      addOn = i18n.ngettext('extension', 'extensions', count);
```

Comment:
Oh, this is clever but unfortunately it's best to put the entire string in here instead. Some locales have complex pluralisation rules and it's best to have the entire string in context. I know it's a bit less elegant that way, but if you look at the example I referenced earlier that's how we do it. Could you put the entire string (in both singular and plural forms) in one gettext call?

Reply:
Yes sure, thanks for letting me know regarding this. Will keep this in mind in future.